File: CSS 112/LAB4/q3.c
#include <stdio.h>
#include <conio.h>
#include <ctype.h>

void even(int arr[], int size, int even_array[], int *even_size);
void odd(int arr[], int size, int odd_array[], int *odd_size);

int main() {
    int che1, che2,arr[20], arri = 0, i, n;
    printf("Input number of Elements (maximum 20) : ");
    scanf("%d", &n);

    printf("Enter all %d Array Elements : \n", n);
    for (i = 0; i < n; ++i) {
        scanf("%d", &arr[i]);
    }

    arri = n;
    int even_array[arri],odd_array[arri],even_size = 0,odd_size = 0;

    even(arr, n, even_array, &even_size);
    odd(arr, n, odd_array, &odd_size);

    printf("\nAll even numbers are: ");
    for (i = 0; i < even_size; ++i) {
        if(i > 0){
            printf(",");
        }
        printf(" %d", even_array[i]);
    }
    printf("\nAll odd numbers are: ");
    for (i = 0; i < odd_size; ++i) {
        if(i > 0){
            printf(",");
        }
        printf(" %d", odd_array[i]);
    }

    return 0;
}

void even(int arr[], int size, int even_array[], int *even_size)
{
    int i;
    for(i = 0; i < size; i++){
        if (arr[i] % 2 == 0 ){
            even_array[(*even_size)++] = arr[i];
        }
    }

}
void odd(int arr[], int size, int odd_array[], int *odd_size)
{
    int i;
    for(i = 0; i < size; i++){
        if (arr[i] % 2 != 0 ){
            odd_array[(*odd_size)++] = arr[i];
        }
    }
}

File: CSS 111/Python/LAB2/Q3.py
# q = int(input())
# prime = []
# is_prime = True
# for i in range(2, q):
#     if q % i == 0:
#         is_prime = False
#         prime.append(i)
#         prime.append(int(q/i))
#         break

# if is_prime:
#     print(q, "is prime")
# else:
#     if len(prime) == 1:
#         print(q, "=", prime[0], "x", prime[0])
#     else:
#         print(q, "=", prime[0], "x", prime[1])


n = int(input())

k = 2
while k <= n:
    if n % k == 0:
        break
    k += 1

if k == n:
    print("prime number")
else:
    print()